NorthstarDLL/engine/r2engine.h

```diff
@@ -201,4 +201,64 @@ namespace R2
 	extern server_state_t* g_pServerState;
 
 	extern char* g_pModName;
+
+	// clang-format off
+	OFFSET_STRUCT(CGlobalVars)
```

Comment:
this whole struct is the main thing cglobals-pr adds, so i think it's safe to say something has gone wrong given its in the diff for this pr

Reply:
Merging main back into this branch resolved diff issues.